src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/PreparedStatementTest.java

```diff
@@ -170,6 +170,8 @@ public void testStatementPooling() throws SQLException {
             }
         }
 
+        Utils.skipTestForSQLServer2008((SQLServerConnection) DriverManager.getConnection(connectionString));
```

Comment:
Instead of skipping the test, we can make it compatible with SQL Server 2008 :)

You can try replacing  `throw`  with `RAISERROR`